test/e2e/tests/contract-interactions.spec.js

```diff
@@ -37,13 +33,13 @@ describe('Deploy contract and call contract methods', function () {
         await driver.clickElement({ text: 'Connect', tag: 'button' });
         await driver.waitUntilXWindowHandles(3);
         await driver.delay(5000);
-        windowHandles = await driver.getAllWindowHandles();
-        extension = windowHandles[0];
-        dapp = await driver.switchToWindowWithTitle(
+        let windowHandles = await driver.getAllWindowHandles();
+        const extension = windowHandles[0];
```

Comment:
Super minor but 

```
const [extension] = windowHandles
```

Reply:
Will clean this up in a separate PR so I don't invalidate these approvals